src/sw.js

```diff
@@ -0,0 +1,32 @@
+self.addEventListener("install", function () {
+  self.skipWaiting();
+});
+
+self.addEventListener("activate", (event) => {
+  event.waitUntil(self.clients.claim());
+});
+
+self.addEventListener("fetch", function (event) {
+  event.respondWith(
+    fetch(event.request)
+      .then(function (response) {
+        // We only need to set the headers for index.html
+        if (!response.url.includes("index.html")) return response;
```

Comment:
At least it appears that we only need the headers for `index.html`. I'm not entirely sure if that's 100% spec compliant though, maybe we should set them for everything.

Reply:
I decided to comment this bit out